src/index.css

```diff
@@ -0,0 +1,116 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  color: white;
+  background-color: rgb(184, 41, 124);
+}
+
+.navBar {
+  width: 100%;
+  padding: 18px;
+  display: flex;
+  flex-direction: row;
+  gap: 40%;
+  border-bottom: 1px solid rgb(0, 0, 0);
+  color: black;
+}
+
+i {
+  font-size: 24px;
+}
+
+.navbarRightDiv {
+  display: flex;
+  flex-direction: row;
+  gap: 24px;
+}
+
+.europeMainDiv {
+  margin: 24px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 24px;
+}
+
+.europeImage {
+  width: 100%;
+}
+
+.europeDatialsDiv {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.CountryListMainDiv {
+  margin-top: 40px;
+  display: flex;
+  flex-direction: column;
+}
+
+.CountryListTitle {
+  align-self: center;
+}
+
+.CountryList {
+  display: grid;
+  width: 100%;
+  grid-template-columns: repeat(2, 1fr);
+  align-items: center;
+  justify-items: center;
+}
+
+.one-country {
+  margin-top: 24px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
- [ ] Please make sure we follow the design, currently your individual countries cards has nothing that conveys a user that the cards are clickable and you can click to get more details please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/219847007-440bf47a-d57c-411c-bdf3-c91f6e6485f9.png)

however in the original design you can see a small arrow symbol at the top right corner :point_down: 
![image](https://user-images.githubusercontent.com/97521557/219847029-6e87634d-3ee3-4e93-a295-754e73994030.png)

ALong with adding a metric , please also consider adding this arrow to make it clear that this takes you to another details page :+1: